utils/helpers.js

```diff
@@ -20,4 +20,7 @@ module.exports = {
       return false;
     }, false);
   },
+  is_approved: (status) => {
```

Comment:
What if we made this an `is_match` helper that expects two variables and can compare them. That way we can reuse it for lots of comparisons. Depends on if my usage example below, with the string literal, works or not.

Usage in template would look like `{{#if (is_match status "approved")}}`. (I'm not 100% positive you can pass in string values like that, but try it out :) )

Reply:
It works! Just added another commit with the change. Thanks.